server/components/index.js

```diff
@@ -0,0 +1,21 @@
+const { sectorInformation } = require("./sectorInformation");
+const { topPeers } = require("./topPeers");
+const { companyOverview } = require("./companyOverview");
+const { latestNewsInterval } = require("./latestNewsInterval");
+const { keyStats } = require("./keyStats");
+const { stockTicker } = require("./stockTicker");
+const { chartData } = require("./chartData");
+const { searchQuery } = require("./searchQuery");
+const { getAllCompanies } = require("./getAllCompanies");
```

Comment:
Consider renaming to `allCompanies`?

Reply:
`allCompanies` sounds like a noun. Like it'd be an array I could import.
`getAllCompanies` would be a function which returns an array of companies.